### DISCUSSION TITLE
Linux Mint (Ubuntu) Flatpak stops the backgorund service when you close it

### DISCUSSION THREAD

**Discussion by DarkLordGMS** [rustdesk/rustdesk#9658] [General]
### Bug Description

Every time I close the RustDesk Flatpak (v1.3.1) on Linux Mint Cinnamon v22 (Kernel 6.8.0-45), the background service ends and therefore it does not allow me to remote into the machine.

### How to Reproduce

1. Install RustDesk flatpak
2. Open RustDesk
3. Try to remote to it from another machine and it works fine
4. Click the X to close it
5. Try to remote to it from another machine and it shows as offline

### Expected Behavior

It should still work since the Service is running but it looks like it stops when you click the X to close it

### Operating system(s) on local side and remote side

Linux Mint v22 Cinnamon - Kernek 6.8.0-45-generic

### RustDesk Version(s) on local side and remote side

1.3.1

### Screenshots

![image](https://github.com/user-attachments/assets/34ec781e-4912-435f-807d-d3a313eb00a1)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/8098
